channel-slack/build.gradle

```diff
@@ -0,0 +1,17 @@
+ext.moduleName = 'com.synopsys.integration.alert.channel.slack'
+
+dependencies {
+    implementation platform(project(':alert-platform'))
+
+    implementation project(':alert-common')
+    implementation project(':api-common-model')
+    implementation project(':api-channel')
+    implementation project(':api-processor')
+    implementation project(':api-descriptor')
+
+    implementation 'org.springframework:spring-web'
```

Comment:
Is this dependency necessary?

Reply:
I added it early on to fix up some tests since it was included in the channels subproject. Just removed it and tests are  still passing so I'll remove this dependency.